src/v0/destinations/hs/util.js

```diff
@@ -626,6 +635,31 @@ const getHsSearchId = (message) => {
   return { hsSearchId };
 };
 
+/**
+ * returns updated traits
+ * @param {*} propertyMap 
+ * @param {*} traits 
+ * @param {*} destination 
+ */
```

Comment:
unit test is missing

Reply:
will add it as backlog, as of now we are going with hotfix